### DISCUSSION TITLE
更新最新版后（2.4.3），zotero无法显示条目了

### DISCUSSION THREAD

**Discussion by yuduchen-han** [MuiseDestiny/zotero-style#241] [Q&A]
大神，今天更新了最新版2.4.3后，我的zotero显示不出来条目了，我把style关掉后可以显示，然后我又打开试了一下，条目就又消失了。
但是，，，刚刚我向关掉style时，zotero附加组件（原来的插件按钮）也打不开了，无法关闭style，条目显示不出来。有什么办法解决吗？

**Comment by yuduchen-han** [comment]
![image](https://user-images.githubusercontent.com/76163471/224620272-3ab46458-1e82-4095-9cb0-7d7a04014f3f.png)
如图，重启多次后仍然显示不出条目

**Reply by MuiseDestiny** [reply]
有输出日志吗

**Reply by yuduchen-han** [reply]
那时候没存。。我一会回去弄一下错误日志

**Comment by MuiseDestiny** [comment]
在后台彻底杀死zotero后重启。估计是更新后没重启导致的

**Reply by PlumPhoebe** [reply]
彻底杀死zotero重启之后还是不显示文献条目，只有关掉style之后才会显示出来

**Reply by yuduchen-han** [reply]
是的，后台杀死包括重启电脑，都无法显示条目。不过我重启电脑后打开zotero，可以打开插件列表关掉style了

**Reply by MuiseDestiny** [reply]
看下输出日志呢

**Reply by MuiseDestiny** [reply]
刚重新上传了，下载安装试试

**Reply by MuiseDestiny** [reply]
首选项忘记写了一个东西

**Reply by MuiseDestiny** [reply]
试了下，就是这个原因，重新下载安装就可以了。抱歉。

**Reply by yuduchen-han** [reply]
抱歉这么晚回复，重新下载安装后问题已经解决，谢谢您啦

**Reply by Gravitya** [reply]
所以是什么问题啊，我也遇到了这个问题，怎么解决？

**Comment by JingWucc** [comment]
**我也是这个问题``呜呜
